linux_os/guide/system/software/sudo/sudo_remove_nopasswd/bash/shared.sh

```diff
@@ -0,0 +1,3 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_debian,multi_platform_ubuntu,multi_platform_ol
+
+sed -i '/^#/!s/\(.*\)[Nn][Oo][Pp][Aa][Ss][Ss][Ww][Dd][:][ ]\(.*\)/\1\2/g' /etc/sudoers /etc/sudoers.d/*
```

Comment:
Actually this NOPASSWD must be only upper case, lower case nopasswd will be refused by visudo.